### PR TITLE
Minor Fixups

### PR DESCRIPTION
**Pull request by terrapin47**
Also fixes all places where someone accidentally put in a space instead of a tab.

### CODE REVIEW

File: QUEST.tsv [hunk 1]
```diff
@@ -179,7 +179,7 @@ QUEST_20150317_000178	NPC HIDE/UNHIDE Setting!
 QUEST_20150317_000179	Follower Algis
 QUEST_20150317_000180	This is the moment the Goddess foretold to the first Paladin. {nl}Come. Ask me anything.
 QUEST_20150317_000181	A long, long time ago, there were nomads that suffered from a cursed plague. {nl}The honorable first Paladin removed the curse with a Holy Relic.
-QUEST_20150317_000182	The Watchers are the descendants of those nomads. {nl}However, the Holy Relic was not meant to be used for healing those people.
+QUEST_20150317_000182	The watchers are the descendants of those nomads. {nl}However, the Holy Relic was not meant to be used for healing those people.
```
Comment (Sunaries) [inline QUEST.tsv]: Why did you uncapitalize it? It's a group. They're capitalized (Hunters, Believers, Followers, Pilgrims) Same for the other lines.

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -7775,7 +7775,7 @@ QUEST_LV_0100_20150730_007770	{memo X}Accroding to Yuta, the corpses came our fr
 QUEST_LV_0100_20150730_007771	Pull out the spell doll from the box of the grave guard
 QUEST_LV_0100_20150730_007772	{memo X}The grave guard, Sigis asked you to pull out the spell doll from the box of the grave guard.
 QUEST_LV_0100_20150730_007773	{memo X}Hand over the spell doll of the grave guard obtained from the box to Sigis.
-QUEST_LV_0100_20150803_007774	$To retrieve the revelation, please go to Sirdgela Forest.{nl}Now is the chance when the Demon Lord, Bramble is recovering its power.
+QUEST_LV_0100_20150803_007774	$To retrieve the revelation, please go to Sirdgela Forest.{nl}Now is the chance when the Demon Lord, Bramble is recovering his power.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Demon Lord, Bramble -> Demon Lord Bramble

File: QUEST.tsv (hunk 1 above)
Comment (terrapin47) [inline QUEST.tsv]: Hmm, it wasn't consistent.  I'll change the other 2 to capitalized as well then.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**